### ISSUE TITLE
Do another audit of accessibility data

### ISSUE THREAD

**Issue by alistairjcbrown** [clusterflick/scripts#35]

**Comment by alistairjcbrown** [comment]
Things to look for:
* Look at transfored data notes and titles to see if there's accessibility information noted which isn't in our accessibility object
* Spot check venues to see if there's any accessibility information missing
